libopenage/console/buf.cpp

```diff
@@ -617,6 +617,7 @@ void Buf::process_escaped_cp(int cp) {
 	switch (cp) {
 	case 'c': //RIS (full reset)
 		this->reset();
+		break;
```

Comment:
I think this is okay, as a full reset should not imply an aborted escape sequence.